### PR TITLE
v0.20

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: _posts/2022-02-22-gleam-v0.19-released.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+JavaScript). Today Gleam [v0.20.0][release] has been released, let's take a look
+at what's been added over the last month.
+
+[release]: https://github.com/gleam-lang/gleam/releases/tag/v0.20.0
+
+
+## Compiling Gleam from WebAssembly
+
+The Gleam compiler itself can now be compiled to and used from WebAssembly. This
+enabled Gleam code to be compiled in various new places, most notably within web
```
Comment (michallepicki) [inline _posts/2022-02-22-gleam-v0.19-released.md:18]:
```suggestion
enables Gleam code to be compiled in various new places, most notably within web
```

File: _posts/2022-02-22-gleam-v0.19-released.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+With this new interface John Doneth has created a [Gleam Playground][playground],
+giving people a way to try out writing Gleam code without having to install
+anything on their computer or set up a development environment.
+
+[playground]: https://johndoneth.github.io/gleam-playground/
+
+How does it work? I'm glad you asked!
+
+The Gleam compiler is split into 2 components, `core` and `cli`, each of which
+is a project written in the Rust programming language. The `core` project has
```
Comment (michallepicki) [inline _posts/2022-02-22-gleam-v0.19-released.md:30]:
```suggestion
is a project written in the Rust programming language. The `core` project is
```

File: _posts/2022-02-22-gleam-v0.19-released.md [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+must have a pattern that matches it or else it could crash.
+
+These values are not matched:
+
+  - Technician
+```
+
+Much better!
+
+Currently the compiler supported a limited form of this exhaustiveness checking
```
Comment (michallepicki) [inline _posts/2022-02-22-gleam-v0.19-released.md:134]:
```suggestion
Currently the compiler supports a limited form of this exhaustiveness checking
```

File: _posts/2022-02-22-gleam-v0.19-released.md [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+Project licence values are now verified as being valid SPDX licences, part of a
+wider BEAM ecosystem goal to validate licences used within the Hex package
+manager. Thank you to [Jonathan Arnett](https://github.com/J3RN) for this
+contribution and for his thorough research into the state of play with BEAM
+package licencing.
+
+The source code formatter's output format has been improved with special cases
+to make some common patterns look better.
+
+And as always, we've fixed some bugs 🐜
```
Comment (michallepicki) [inline _posts/2022-02-22-gleam-v0.19-released.md:181]: There's a second similar sentence when linking to the changelog a few lines below. Maybe that one could be moved here?

**Review by michallepicki — COMMENTED** [review]
😍
What do you think about mentioning example-todomvc and gleam-templates? Maybe a one sentence paragraph in the "And the rest" section? On the other hand templates a few some kinks to iron out, and maybe the example todomvc could even have its own blog post when ready and deployed...

**Comment by lpil** [comment]
@michallepicki 
> What do you think about mentioning example-todomvc and gleam-templates? Maybe a one sentence paragraph in the "And the rest" section? On the other hand templates a few some kinks to iron out, and maybe the example todomvc could even have its own blog post when ready and deployed...

Once they're ready I'd like to give them their own dedicated post, similar to [this one](https://dev.to/rtfeldman/tour-of-an-open-source-elm-spa). My theory is that more regular content will help build some hype.

**Pull request merged**